lib/SILOptimizer/Utils/OptimizerStatsUtils.cpp

```diff
@@ -276,15 +276,22 @@ llvm::cl::opt<std::string> StatsOnlyFunctionsNamePattern(
 struct FunctionStat {
   int BlockCount = 0;
   int InstCount = 0;
+  /// True when the FunctionStat is created for a SIL Function after a SIL
+  /// Optimization pass has been run on it. When it is a post optimization SIL
+  /// Function, we do not want to store anything in the InlinedAts Map in
+  /// InstCountVisitor.
+  bool NewFunc = false;
```

Comment:
Based on your comment the flag indicates whether we are pre- or post-pass? It's still not clear the variable is called NewFunc then?

Reply:
This is based on the convention that is used throughout the code:

if we look at line 1047 in the same file.

```
auto &OldFuncStat = FuncStat;
FunctionStat NewFuncStat(F, true);
```

`NewFuncStat` is a `FunctionStats` object for a function post-pass, `OldFuncStat` is for a for a function pre-pass. The flag is there to denote to the `FunctionStat` object, that the function it is created with is post-pass if true, I can change the name to `PrePass` or `PostPass` if that makes it clearer?

Reply:
If it's a convention in the file, I have no concerns.